### ISSUE TITLE
Suggestion: Multiple Knockback Profiles

### ISSUE THREAD

**Issue by c0dingnoobi** [hpfxd/PandaSpigot#28]
Straightforward as title says.  Might also be useful to set those knockback profiles per player (mostly to support something like practice servers with different types of gamemodes f.e combo/speed/nospeed).

**Comment by uRyanxD** [comment]
> was this implemented?

Yes, but not within PandaSpigot, the plugin https://github.com/hpfxd/PandaKnockback implements knockback profiles per player
